navigator_engine/cli.py

```diff
@@ -13,7 +13,7 @@ def navigator_engine():
         pass
 
     @navigator_engine.command()
-    @click.argument('graph-config-file', default='Estimates 22 BDG [Final].xlsx')
+    @click.argument('graph-config-file', default=app.config.get('TEST_DATA_SPREADSHEET'))
```

Comment:
Would suggest renaming the config param and also putting the old default value back in if the config param isn't defined. 

```suggestion
    @click.argument(
        'graph-config-file', 
        default=app.config.get(
            'DEFAULT_DECISION_GRAPH', 
            'Estimates 22 BDG [Final].xlsx'
        )
    )
```